docs/src/main/resources/microsite/data/menu.yml

```diff
@@ -0,0 +1,60 @@
+options:
+
+    - title: Setup PureConfig
+      url: setup-pureconfig.html
+
+    - title: Use Pureconfig
```

Comment:
In order to show the currently selected item as active in the sidebar, this needs to match the `page.title` variable. There are a couple of pages here that don't match. We can always display a different title in the first heading of the page.

Reply:
Thanks @jcazevedo for checking this, I didn't notice. It should be fixed with 76d759c.